File: RedSheet.DbEntities/Models/vLanguage.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vLanguages", Schema = "dbo")]
	[Serializable]
	public partial class vLanguage  
	{
 
	[Required]
	public bool Active { get; set; } 
 
	[Required] 
	[MaxLength(2)]
	public string LanguageCode { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Key]
	public int LanguageId { get; set; } 
 
	[Required] 
	[MaxLength(100)]
	public string LanguageName { get; set; } 

	public vLanguage()
	{
    }

    }
}

File: redsheet-portal/src/app/database-models/v-language.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vLanguage {
    constructor(vLanguage?: vLanguage )  {
        let properties = [ "active", "languageCode", "languageId", "languageName",];
        for (let property of properties)
            if (vLanguage && vLanguage[property])
                this[property] = vLanguage[property];
    }
 
	active : boolean = false ;
 
    @required()
    @maxLength(2)
	languageCode : string =   undefined;
 
	languageId : number =   0 ;
 
    @required()
    @maxLength(100)
	languageName : string =   undefined;


}
